src/helpers/Mail/templates/notifications.js

```diff
@@ -0,0 +1,6 @@
+export default (data) => {
+  const message = {};
```

Comment:
You can do this:  const message = {
    subject: 'Authors Haven - Notification',
    html: `<p>${data.message}</p>`
}